### PR TITLE
INBOX: Ability to drag item from INBOX into DRAFT #371

### PR DESCRIPTION
**Pull request by ArindamReact**
I've done the following changes to this 
1. added drag function and hand icon in Inbox, also passed the data from props to home page
2. In this I've also added one function regarding removing items from the inbox after being dragged to draft
3. made the changes to a home component in which the main logic is written.

### CODE REVIEW

**Comment by ArindamReact** [comment]
This PR is for both tasks
1. INBOX: Once an item has been dragged from INBOX to DRAFT, remove from DRAFT automatically #370
2. INBOX: Ability to drag item from INBOX into DRAFT #371

**Comment by mrjamieeast** [comment]
This doesn't work for me.
It deletes the lines, but doesn't put them in DRAFT. Please see screen recording

https://user-images.githubusercontent.com/104105362/176830099-29ccb9b0-5015-4c77-9cbd-14efc5fe3340.mov

.

File: package.json [hunk 1]
```diff
@@ -20,8 +20,8 @@
     "@headlessui/vue": "1.6.5",
     "@heroicons/vue": "1.0.6",
     "@types/linkify-it": "^3.0.2",
-    "@vueuse/core": "8.7.5",
-    "firebase": "9.8.4",
+    "@vueuse/core": "8.6.0",
+    "firebase": "9.8.3",
```
Comment (chrisns) [inline package.json:24]: why are you downgrading packages?

Comment (ArindamReact) [reply package.json:24]: we forget to mark pr in draft and we were checking some dependencies rebuild, That's why package json show changes

**Pull request closed without merging**